src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,90 +1,49 @@
 <div *ngIf="contentData">
   <div class="row">
     <!-- HEALTH -->
-    <div class="col-md-6">
-      <div class="well">
-        <fieldset>
-          <legend i18n>Health</legend>
-          <ng-container i18n>Overall status: </ng-container>
-          <span [ngStyle]="contentData.health.status | healthColor">{{ contentData.health.status }}</span>
-          <ul>
-            <li *ngFor="let check of contentData.health.checks">
-              <span [ngStyle]="check.severity | healthColor">{{ check.type }}</span>: {{ check.summary.message }}
-            </li>
-          </ul>
-        </fieldset>
-      </div>
-    </div>
+    <cd-info-card title="Health"
+                  class="col-md-6">
+      <hr class="media-underline" />
+      <ng-container i18n>Overall status: </ng-container>
+      <span [ngStyle]="contentData.health.status | healthColor">{{ contentData.health.status }}</span>
+      <ul>
+        <li *ngFor="let check of contentData.health.checks">
+          <span [ngStyle]="check.severity | healthColor">{{ check.type }}</span>: {{ check.summary.message }}
+        </li>
+      </ul>
+    </cd-info-card>
 
     <div class="col-md-6">
       <!--STATS -->
       <div class="row">
-        <div class="col-md-6">
-          <div class="well"
-               *ngIf="contentData.mon_status">
-            <div class="media">
-              <div class="media-left">
-                <i class="fa fa-database fa-fw"></i>
-              </div>
-              <div class="media-body">
-                <span class="media-heading"
-                      i18n="ceph monitors">
-                  <a routerLink="/monitor/">Monitors</a>
-                </span>
-                <span class="media-text">{{ contentData.mon_status | monSummary }}</span>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="col-md-6">
-          <div class="well"
-               *ngIf="contentData.osd_map">
-            <div class="media">
-              <div class="media-left">
-                <i class="fa fa-hdd-o fa-fw"></i>
-              </div>
-              <div class="media-body">
-                <span class="media-heading"
-                      i18n="ceph OSDs">
-                  <a routerLink="/osd/">OSDs</a>
-                </span>
-                <span class="media-text">{{ contentData.osd_map | osdSummary }}</span>
-              </div>
-            </div>
-          </div>
-        </div>
+        <cd-info-card title="MONITORS"
+                      class="col-md-6"
```

Comment:
If `col-md-6` seems to be the default one, why not overriding it, or setting it as default:
```javascript
@Input() class: string = 'col-md-6';
```

Reply:
I'll implement it.
By setting it at selector level I was trying to avoid an additional enclosing <div> (needed to respect the current look and feel).
It would also be good to define if the project style guide contains "Convention over configuration" (e.g. providing sensible default values as much as possible)
in order for all contributors to follow the same coding style.

Reply:
Love that 'convention over configuration'! Not sure if there's a style guide for Ceph (only could find [this](https://github.com/ceph/ceph/blob/master/CodingStyle)). This might be an interesting topic to discuss!